src/components/InputPassword/InputPassword.css

```diff
@@ -0,0 +1,14 @@
+.InputPassword{
+    width: 400px;
+    height: 72px;
+    border: 2px solid #fff;
+    box-sizing: border-box;
+    background-color: #008ac1;
+    color: #fff;
+    font-size: 16px;
```

Comment:
make them inherit the font-size so that they take on whatever has been defined by the parent.
`font-size: inherit`